setup.cfg

```diff
@@ -30,22 +36,45 @@ universal=1
 zip_safe = false
 py_modules = multicast
 test_suite = tests
-python_requires = >=3.7, !=3.11.*
+python_requires = >=3.7, !=3.13.*
 setup_requires =
-    setuptools>=45.0.0
-    wheel>=0.37.0
+  setuptools>=45.0.0
+  wheel>=0.37.0
+  build>=1.2.1
 
 scripts =
   multicast/__main__.py
+  setup.py
 packages = find:
 
 
 [options.packages.find]
 where =
-    multicast/
-    tests/
-    *.py
+  multicast/
+  tests/
+  *.py
 include =
-    multicast
+  multicast
 exclude =
-    tests
+  tests
+
+[options.extras_require]
+testing =
+  # upstream
+  pytest >= 7
+  pytest-checkdocs >= 2.4
```

Comment:
❓ is this used in CI ?

Reply:
:x: nope!
this is not consistent with `requirements.txt` so nothing in CI actually makes use of it at this time.

Reply:
Opened ticket #60 🏷️ for this.